mpba/mpba/parser.py

```diff
@@ -25,7 +25,14 @@ def sum_up_from(values):
         if key != "total" and value is not None
     ])
 
-
+#Infere o tipo de funcionário a partir do seu cargo.
+def get_func_type(cargo):

```

Comment:
Não devemos abrir o arquivo e passar para cada funcionário. Munda esse mapa reader somente no início do programa e depois faz apenas um get no mapa que já está em memória.